lib/atomic/organizations/organization.ex

```diff
@@ -7,14 +7,15 @@ defmodule Atomic.Organizations.Organization do
   alias Atomic.Organizations.{Announcement, Board, Card, Department, Membership, Partner}
   alias Atomic.Uploaders
 
-  @required_fields ~w(name description)a
+  @required_fields ~w(name handle description)a
```

Comment:
```suggestion
  @required_fields ~w(name slug description)a
```
I think `handler` is the term you are looking for. Nevertheless, since `handler` is somewhat of a reserved key word (`handle_event`, `handle_params`, etc) and may cause some confusion, what do you thnk about using `slug` instead? :bulb: 

The correct term on the FE should be something like `username`

Reply:
Done , thanks for the alert !! :+1: